### PR TITLE
added batch migrations and renamed functions

### PR DESCRIPTION
**Pull request by JGcarv**

### CODE REVIEW

File: contracts/Tellor.sol [hunk 1]
```diff
@@ -81,12 +81,46 @@ contract Tellor is TellorStake {
...
+     * @param _amount iis the amount to mint to the user
+     */
+    function _migrateFrom(
+        address _origin,
+        address _destination,
+        uint256 _amount
+    ) internal {
         require(!migrated[_origin], "already migrated");
         _doMint(_destination, _amount);
         migrated[_origin] = true;
```
Comment (themandalore) [inline contracts/Tellor.sol:126]: I think we have to take this out for the dexBlue case...can you add a function to lock it or unlock it once we're done with the migration

Comment (themandalore) [reply contracts/Tellor.sol:126]: and with this, if you take this part out, you can just remove the migrateFrom and all can go through the migrateFor

Comment (JGcarv) [reply contracts/Tellor.sol:126]: I honestly think that this is adding too much logic to tellor. My solution would be to migrate all dexBlue balance to another account, or contract, and then do the distribution to holders from there(it'll be probably cheaper too)

This way we can keep a strict rule that no address migrates twice

Comment (themandalore) [reply contracts/Tellor.sol:126]: Perfect.

**Review by themandalore — APPROVED** [review]

**Pull request merged**